### ISSUE TITLE
3. Kaarten missen elementen met toegankelijke namen (API's template A11y)

### ISSUE THREAD

**Issue by TessaViergever** [frameless/developer.overheid.nl#111]
### Betrokken pagina's

[Zoek API's](https://rijkshuisstijl-community-templates-3losjges9-nl-design-system.vercel.app/developer-overheid/api-overzicht)

### Beschrijf het probleem

De Kaarten missen elementen met toegankelijke namen. De Koptekst, Organisatie en de pilvormige API-type labels zijn momenteel niet focusbaar en hebben geen toegankelijke naam. Dit betekent dat de focusvolgorde van de "+ API toevoegen" knop helemaal naar beneden springt naar de Footer, wat onverwacht is.

### Relevante code

```html
<div class="rhc-card__content"><div class="rhc-card__icon"></div><div class="rhc-card__heading">CBS Cijfers Open Data</div><div class="rhc-card__description">Centraal Bureau voor de Statistiek.</div><div class="rhc-card__metadata">Alle Nederlandstailige Statistieken van CBS ontsloten via oData versie 4</div><div class="utrecht-badge-list"><span class="utrecht-data-badge rhc-templates-databadge">OData</span></div></div>
```

### WCAG-richtlijn

WCAG 2.4.3 Focus Order + 4.1.2 Name, Role, Value, 2.1.1 Keyboard Level A

### Voeg een afbeelding toe (indien relevant)

![A widget shows the accessible names of the elements on the page. The Card elements have no accessible names.](https://github.com/user-attachments/assets/e736a52e-b3e3-4b35-a78c-1e0e491866b8)

### Referentienummer

TBD
